### ISSUE TITLE
Should raise an error when module-alias cannot be found

### ISSUE THREAD

**Issue by tals** [tleunen/eslint-import-resolver-babel-module-alias#13]
in `getMappingFromBabel()`, you perform the lookup with:
`const pluginConfig = c.config.plugins.find(p => p[0] === 'module-alias');`

if the key cannot be found (babel for example also allows `babel-plugin-module-alias`), you'll end up with cryptic errors from eslint:
```
error  Resolve error: Cannot read property '1' of undefined                    import/no-unresolved
```

which is the user now has to track down.

**Comment by tleunen** [comment]
Good catch @tals! Want to make a PR to also support the full name? And add a check for the plugin to fix this potential error 1 of undefined?

No worries if you don't have time, I'll do it tomorrow in that case.

**Comment by tleunen** [comment]
Fixed in 1.5.0. Thanks @tals and sorry for the long long delay :/
